### PR TITLE
frontend: add table and modal css style

### PR DESCRIPTION
**Pull request by IrynaKamenska**

### CODE REVIEW

File: frontend/src/CreateBook.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React, {FormEvent} from "react";
 import Modal from 'react-modal';
 import axios from "axios";
 import {BookState} from "./BookState";
+import "./css/BookOverview.css";
```
Comment (ChristianBorn) [inline frontend/src/CreateBook.tsx:5]: If you need the css from your BookOverview component for the CreateBook component consider creating a new css-file for CreateBook with only the css, that is used there

File: frontend/src/CreateBook.tsx [hunk 2]
```diff
@@ -48,11 +49,11 @@ export default function CreateBook(props: ModalProps) {
     }
 
     return (
-        <Modal
-            isOpen={props.modalIsOpen}
-            onRequestClose={props.closeModal}
-            contentLabel="Example Modal"
-            ariaHideApp={false}
+        <Modal className="modal"
```
Comment (ChristianBorn) [inline frontend/src/CreateBook.tsx:52]: This removes the styles that the React modal already brings with it. I found them useful for a start.
Consider removing the class and setting custom styles with the class:
.ReactModal__Content--after-open

File: frontend/src/UpdateBook.tsx [hunk 3]
```diff
@@ -96,13 +97,12 @@ function UpdateBook(props: UpdateBookProps) {
                     <option value={BookState.NOT_AVAILABLE}>NOT_AVAILABLE</option>
                 </select>
                 <br/><br/>
-                <button>Update</button>
+                <div className="modal-body">
+                    <h5>Are you sure to update this book?</h5>
```
Comment (ChristianBorn) [inline frontend/src/UpdateBook.tsx:101]: Semantically, in order to have a h5, there have to be h1-h4 before that.
Consider using a different heading or a \<p> with styling

**Review by ChristianBorn — CHANGES_REQUESTED** [review]

**Comment by ChristianBorn** [comment]
Code smell should be fixed, the rest of my comments is optional

**Review by Marc120985 — APPROVED** [review]

**Review by ChristianBorn — APPROVED** [review]

**Pull request merged**